Microsoft.Toolkit.Uwp.UI.Controls/TextToolbar/Symbols/Symbol.cs

```diff
@@ -53,10 +53,22 @@ private void CheckIsEnabled()
 
         ~Symbol()
         {
-            Dispatcher?.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Normal, () =>
+            try
             {
-                isEnabledWatcher?.Dispose();
-            });
+                Dispatcher?.RunAsync(Windows.UI.Core.CoreDispatcherPriority.Normal, () =>
+                {
+                    try
```

Comment:
Curious what exceptions were happening here.

Reply:
I was getting weird exceptions when the control was getting unloaded, but only in Release mode.